README.md

```diff
@@ -12,12 +12,18 @@ As Hypervisor the following systems are supported:
 
 - List existing snapshots of a virtual machine.
 - Create a snapshot of a virtual machine.
-- Revert existing virtual machine to a previously created snapshot.
+- Revert existing virtual machines to a previously created snapshot.
 - Remove existing snapshots of a virtual machine.
 
 ## Installation
 
-See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin) for how to install Foreman plugins
+This plugin installation is not supported by the foreman-installer, you need to install it manually:
+
+```sh
+dnf install rubygem-foreman_snapshot_management
```

Comment:
Can you also install it on Foreman on Debian? If so, what's the package name?

Reply:
This is not SCC manager but snapshot management and it is not dependent on Katello but still does not exist for Debian afaik.